### PR TITLE
DR-1504 Add logging when creating a google project

### PR DESCRIPTION
**Pull request by nmalfroy**
Logs the request body before TDR attempts to ask Google to create a project

### CODE REVIEW

**Review by snf2ye — APPROVED** [review]

File: src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java [hunk 1]
```diff
@@ -152,6 +152,7 @@ private GoogleProjectResource newProject(
                 .setName(requestedProjectId)
                 .setProjectId(requestedProjectId)
                 .setParent(parentResource);
+        logger.info("creating project with request: {}", requestBody);
```
Comment (fboulnois) [inline src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java:155]:
```suggestion
        logger.info("creating project `{}` with request: {}", requestBody.toPrettyString(), requestBody);
```

Maybe use `toPrettyString`?

**Review by fboulnois — APPROVED** [review]
Suggestion from @snf2ye  :)

File: src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java (hunk 1 above)
Comment (nmalfroy) [reply src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java:155]: Though I will point out that `.toPrettyString()` throws an `IOException` :(

Comment (nmalfroy) [reply src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java:155]: Just checked and both toString and toPrettyString seem to print the same thing

**Review by pshapiro4broad — APPROVED** [review]

**Pull request merged**